po/pt.po

```diff
@@ -235,103 +245,461 @@ msgstr "asp devolveu {}"
+
+#: src/help.rs:66
+msgid ""
+"    --[no]combinedupgrade  Refresh then perform the repo and AUR upgrade "
+"together"
+msgstr ""
+"    --[no]combinedupgrade  Recarrega e depois realiza a atualização do "
+"repositório e AUR em conjunto"
+
+#: src/help.rs:67
+msgid ""
+"    --[no]batchinstall     Build multiple AUR packages then install them "
+"together"
+msgstr ""
+"    --[no]batchinstall     Compila vários pacotes AUR e depois instala-os em "
+"conjunto"
+
+#: src/help.rs:68
+msgid ""
+"    --[no]provides         Look for matching providers when searching for "
+"packages"
+msgstr ""
+"    --[no]provides         Procura fornecedores correspondentes ao procurar "
+"pacotes"
+
+#: src/help.rs:69
+msgid "    --[no]devel            Check development packages during sysupgrade"
+msgstr ""
+"    --[no]devel            Verifica pacotes de desenvolvimento durante o "
+"sysupgrade"
+
+#: src/help.rs:70
+msgid ""
+"    --[no]installdebug     Also install debug packages when a package "
+"provides them"
+msgstr ""
+"    --[no]installdebug     Também instala pacotes de depuração quando um "
+"pacote os fornece"
+
+#: src/help.rs:71
+msgid ""
+"    --[no]sudoloop         Loop sudo calls in the background to avoid timeout"
+msgstr ""
+"    --[no]sudoloop         Cicla invocações sudo em segundo plano para "
+"evitar tempo limite"
+
+#: src/help.rs:72
+msgid "    --[no]chroot           Build packages in a chroot"
+msgstr "    --[no]chroot           Compila pacotes em chroot"
+
+#: src/help.rs:73
+msgid "    --[no]sign             Sign packages with gpg"
+msgstr "    --[no]sign             Assina pacotes com gpg"
+
+#: src/help.rs:74
+msgid "    --[no]signdb           Sign databases with gpg"
+msgstr "    --[no]signdb           Assine base de dados com gpg"
+
+#: src/help.rs:75
+msgid "    --localrepo            Build packages into a local repo"
+msgstr "    --localrepo            Compila pacotes num repo local"
+
+#: src/help.rs:76
+msgid ""
+"    --nocheck              Don't resolve checkdepends or run the check "
+"function"
+msgstr ""
+"    --nocheck              Não resolve checkdepends nem executa a função de "
+"verificação"
+
+#: src/help.rs:77
+msgid ""
+"    --develsuffixes        Suffixes used to decide if a package is a devel "
+"package"
+msgstr ""
+"    --develsuffixes        Sufixos utilizados para decidir se um pacote é um "
+"pacote de desenvolvimento"
+
+#: src/help.rs:78
+msgid ""
+"    --bottomup             Shows AUR's packages first and then repository's"
+msgstr ""
+"    --bottomup             Mostra primeiro os pacotes do AUR e depois os do "
+"repositório"
+
+#: src/help.rs:79
+msgid ""
+"    --topdown              Shows repository's packages first and then AUR's"
+msgstr ""
+"    --topdown              Mostra primeiro os pacotes do repositório e "
+"depois os do AUR"
+
+#: src/help.rs:81
+msgid "show specific options:"
+msgstr "mostrar opções específicas:"
```

Comment:
show is the name of the arg and not just a word.

Reply:
@Morganamilo What do you mean? It shouldn't be translated, is that it?